changelog/issue-#2268.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: patch
```

Comment:
Since this is just a change to documentation, I think `silent` here will do

Reply:
Alright. I will update it.